bin/schemaToInterface.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-env node */
+
```

Comment:
Add a comment here that this file builds a skeleton API for `study` from `schema.json`? What is the output file called?